package.json

```diff
@@ -60,7 +60,7 @@
     "grpc-tools": "1.7.3",
     "grpc_tools_node_protoc_ts": "2.5.0",
     "jsverify": "0.8.4",
-    "mocha": "6.1.3",
+    "mocha": "^6.1.4",
```

Comment:
Please upgrade the dependency using `npm i -DE` to make sure you are using an exact dependency. This, coupled with the lock file, ensure reproducible builds.

Reply:
okay froze version and rebased -- it's running again now

Reply:
:pray: